### DISCUSSION TITLE
Can I install MiniDexed without formatting my Rpi3b?

### DISCUSSION THREAD

**Discussion by agusting** [probonopd/MiniDexed#566] [Q&A] [answered]
Hi, I'd like to run MiniDexed without having to format my Rpi3b. The reason is that I use my Rpi for other things (Pihotle, Model, etc) and if possible I'd like to keep it that way. 
I've searched but all the tutorials I find asume the install on a fresh SD Card.
Is there any way to achieve what I want? 
Thanks in advance.

**Comment by probonopd** [comment, marked as answer]
Hello. You need a dedicated microSD card for MiniDexed because it doesn't run a Linux kernel.

If you absolutely must re-use the SD card, you could:
* Create a "Linux" folder on the boot partition
* Move everything you have so far into that folder
* Put the MiniDexed files into the root direcotry of the boot partition

Then, when you want to run Linux again, you would:
* Create a "MiniDexed" folder
* Move everything (except the "Linux" folder) into the "MiniDexed" folder
* Move the contents of the "Linux" folder into the root directory of the boot volume

Not sure whether it is worth the hassle, given that a decent microSD card can be had for much less than 10 USD.

**Reply by agusting** [reply]
Thanks!
